layers/meta-tegra-support/recipes-test/tegra-tests/vpi1-tests_1.0.bb

```diff
@@ -0,0 +1,25 @@
+DESCRIPTION = "Scripts for testing Tegra VPI samples"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+
+SRC_URI = "\
+    file://run-vpi1-tests.sh.in \
+"
+
+COMPATIBLE_MACHINE = "(tegra)"
+
+S = "${WORKDIR}"
+B = "${WORKDIR}/build"
+
+do_compile() {
+    sed -e's,@SOC_FAMILY@,${SOC_FAMILY},g' \
+        ${S}/run-vpi1-tests.sh.in >${B}/run-vpi1-tests.sh
+}
+
+do_install() {
+    install -d ${D}${bindir}
+    install -m 0755 ${B}/run-vpi1-tests.sh ${D}${bindir}/run-vpi1-tests
+}
+
+PACKAGE_ARCH = "${TEGRA_PKGARCH}"
```

Comment:
If you change the script to detect the SoC type at runtime, you can leave this line as-is.  Otherwise, since the build is now per-SoC, you should change this to `${SOC_FAMILY_PKGARCH}`, since the package is SoC-dependent.

Reply:
Agree